product_docs/docs/mongo_data_adapter/5/06_features_of_mongo_fdw.mdx

```diff
@@ -96,3 +96,58 @@ Steps for retrieving the document:
     { "_id" : { "$oid" : "58a1ebbaf543ec0b9054585a" }, "warehouse_id" : 2, "warehouse_name" : "Laptop", "warehouse_created" : { "$date" : 1447229590000 } }
     (2 rows)
     ```
+
+## Accessing sub-json
+
+MongoDB Foreign Data Wrapper supports retrieving sub-json, by allowing you to retrieve sub-fields from a document.
+You can retrieve sub-fields in a collection residing in MongoDB Foreign Data Wrapper as explained in the following example:
+
+```text
+db1> db.test_sub_json.find()
+[
+  {
+    _id: ObjectId('658040214890799d6e0173d0'),
+    key1: 'hello',
+    key2: {
+      subkey21: 'hello-sub1',
+      subkey22: 'hello-sub2',
+      subtstmp: ISODate('2022-12-16T19:16:17.801Z')
+    }
+  }
+]
+```
+
+Steps for retrieving sub-fields from the document:
+
+1.  Create a forign table. To access a sub-field use the dot (".") in the columnn name as shown below:
+
+```sql
+CREATE FOREIGN TABLE ft_nested_json_test(
+    _id NAME,
+    key1 varchar,
+    "key2.subkey21"   varchar,
+    "key2.subkey22"   varchar,
+    "key2.subtstmp"   timestamp
+)SERVER mongo_server
+OPTIONS (database 'db1', collection 'test-sub_json');
```

Comment:
The 'collection' name should be 'test_sub_json'?

Reply:
Got it.